server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixAccountEntity.java

```diff
@@ -0,0 +1,30 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import jdk.jfr.Enabled;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "netflix_account_entity")
```

Comment:
La table dans le schéma s'appelle `netflix_account` et pas `netflix_account_entity`
```suggestion
@Table(name = "netflix_account")
```